corelib-definitions/src/main/java/eu/europeana/corelib/definitions/edm/entity/Timespan.java

```diff
@@ -14,14 +14,43 @@
 	/**
 	 * Retrieve the edm:begin field of a Timespan
 	 * 
-	 * @return Stringrepresenting the edm:begin field of a timespan
+	 * @return String representing the edm:begin field of a timespan
+	 */
+	
+	String getBeginString();
```

Comment:
All these methods, which are required for the Timespan serialization (jsonld, or xml) must be added (if not available) to eu.europeana.entity.definitions.model.Timespan interface and not to the corelib interface. Please revert back changes in this class.

Reply:
I have reverted the Timespan interface class and implemented the required changes in the entity-related interface.